package.xml

```diff
@@ -17,6 +17,7 @@
   <depend>pybind11_vendor</depend>
   <depend>shared_memory</depend>
   <depend>mpi_cmake_modules</depend>
+  <depend>pam_configuration</depend>
```

Comment:
Unless the config files are needed at build-time `<exec_depend>` should be enough.

Reply:
indeed